frontend/src/lib-common/generated/api-types.d.ts

```diff
@@ -3263,6 +3286,11 @@ export interface Coordinate {
 */
 export type ScheduledJob = 'CancelOutdatedTransferApplications' | 'DvvUpdate' | 'EndOfDayAttendanceUpkeep' | 'EndOutdatedVoucherValueDecisions' | 'FreezeVoucherValueReports' | 'KoskiUpdate' | 'RemoveOldAsyncJobs' | 'RemoveOldDaycareDailyNotes' | 'RemoveOldDraftApplications' | 'SendPendingDecisionReminderEmails' | 'VardaUpdate' | 'InactivePeopleCleanup' | 'InactiveEmployeesRoleReset'
 
+/**
+* Generated from fi.espoo.evaka.shared.security.Action.Child
+*/
+export type Child = 'fi.espoo.evaka.shared.security.Action$Child.READ' | 'fi.espoo.evaka.shared.security.Action$Child.READ_ADDITIONAL_INFO' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_ADDITIONAL_INFO' | 'fi.espoo.evaka.shared.security.Action$Child.READ_APPLICATION' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_ASSISTANCE_ACTION' | 'fi.espoo.evaka.shared.security.Action$Child.READ_ASSISTANCE_ACTION' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_ASSISTANCE_NEED' | 'fi.espoo.evaka.shared.security.Action$Child.READ_ASSISTANCE_NEED' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_BACKUP_CARE' | 'fi.espoo.evaka.shared.security.Action$Child.READ_BACKUP_CARE' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_BACKUP_PICKUP' | 'fi.espoo.evaka.shared.security.Action$Child.READ_BACKUP_PICKUP' | 'fi.espoo.evaka.shared.security.Action$Child.READ_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.DELETE_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.READ_PLACEMENT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_FAMILY_CONTACTS' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_FAMILY_CONTACT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_GUARDIANS' | 'fi.espoo.evaka.shared.security.Action$Child.READ_FEE_ALTERATIONS' | 'fi.espoo.evaka.shared.security.Action$Child.READ_CHILD_RECIPIENTS' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_CHILD_RECIPIENT' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_VASU_DOCUMENT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_VASU_DOCUMENT'
```

Comment:
🤔

Reply:
huh.. thats a bug

Reply:
fixed

Reply:
Or is it?

```
     permittedActions: Child[]
```

```
/**
* Generated from fi.espoo.evaka.attendance.Child
*/
export interface Child {
```

:thinking:

Reply:
asdf

Reply:
Hmm this needs more fixing, need to somehow make sure that we don't confuse classes with same name.. I'll just exclude the classes using permittedActions for now to get master build working

Reply:
Not sure if we should revert the whole thing for now